.github/CODEOWNERS

```diff
@@ -8,3 +8,6 @@
 # These folks own any files in the .github directory at the root of
 # the repository and any of its subdirectories.
 /.github/ @dav3r @felddy @jasonodoom @jsf9k @mcdonnnj
+
+flake.*  @jasonodoom
+*.nix @jasonodoom
```

Comment:
Alphabetize and add a comment:
```suggestion
# Our Nix and flake expert(s) own these files.
*.nix @jasonodoom
flake.*  @jasonodoom
```

Reply:
Why two spaces after `flake.*`, @dav3r?

Reply:
Thanks- that was an oversight.  I just copied/pasted what @jasonodoom had originally.

Reply:
@dav3r sorry, yes I saw @jsf9k email, noticed the spaces and fixed it in git but should have opened in GH to see your suggestions.

Reply:
Looks good in https://github.com/cisagov/awssh/pull/27/commits/449c82e4df23dfe8c8745e341143bd4e8f7a8d1a. 👍

Reply:
@jsf9k I'm resolving this one.  Please re-open if anything has been missed.